cmd/snaptel/flags.go

```diff
@@ -35,21 +35,24 @@ var (
 		Value:  "http://localhost:8181",
 	}
 	flAPIVer = cli.StringFlag{
-		Name:  "api-version, a",
-		Usage: "The Snap API version",
-		Value: "v1",
+		Name:   "api-version, a",
+		Usage:  "The Snap API version",
+		EnvVar: "SNAP_API_VERSION",
+		Value:  "v1",
 	}
 	flSecure = cli.BoolFlag{
-		Name:  "insecure",
-		Usage: "Ignore certificate errors when Snap's API is running HTTPS",
+		Name:   "insecure",
+		Usage:  "Ignore certificate errors when Snap's API is running HTTPS",
+		EnvVar: "SNAP_INSECURE",
 	}
 	flRunning = cli.BoolFlag{
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
 	flPassword = cli.BoolFlag{
-		Name:  "password, p",
-		Usage: "Password for REST API authentication",
+		Name:   "password, p",
+		Usage:  "Password for REST API authentication",
+		EnvVar: "SNAP_PASSWORD",
```

Comment:
@rashmigottipati - do you have the same feeling like me that it might be confusing? I know that this is not related to this PR, but maybe simple changing the usage description to 'Require password for REST API authentication'  or sth like that will be more clear for users that this is a boolean type.  Also, the name of env variable "SNAP_PASSWORD" might suggest that some string (password) should be set there.  Maybe renaming that to "SNAP_REQUEST_PASSWORD" will help to avoid such mistakes. 

This is the only light suggestion. If you agree - that is great, if not - that is also ok for me.